fuse/data/ops/ops_common.py

```diff
@@ -351,6 +353,46 @@ def __call__(self, sample_dict: NDict, op_id: Optional[str], keep_keypaths:List[
         sample_dict = NDict()
         for k in keep_keypaths:
             sample_dict[k] = prev_sample_dict[k]
+
+        return sample_dict
+
+
+class OpMap(OpBase):
```

Comment:
just a thought - "map" became associated with running a set of elements through a provided function.

Mabye OpDictionaryLookup or OpLookup ? 

feel free to ignore if you don't agree.